pii_recognition/registration/bind.py

```diff
@@ -0,0 +1,17 @@
+from pii_recognition import recognisers
+from pii_recognition.registration import registry
+
+
+def _get_default_recognisers():
+    return [
+        recognisers.crf_recogniser.CrfRecogniser,
+        recognisers.first_letter_uppercase_recogniser.FirstLetterUppercaseRecogniser,
+        recognisers.flair_recogniser.FlairRecogniser,
+        recognisers.spacy_recogniser.SpacyRecogniser,
+        recognisers.stanza_recogniser.StanzaRecogniser,
+    ]
+
+
+def _run():
+    for item in _get_default_recognisers():
+        registry.recogniser.add_item(item)
```

Comment:
I think we need a more meaningful name for this function. Maybe `_register_recognisers`?
other than this I think this PR is good to go.

Reply:
That a much better name! I will update it.